### PR TITLE
avatars and profile ui changes, and other stuff

### PR DESCRIPTION
**Pull request by isabelleleeson**
I tested as much as I could but can't test creating accounts cause no qr code scanning on emulator so plz let's test this as quickly as possible tomorrow

image
image
image
image

### CODE REVIEW

File: android/app/src/main/java/com/chorestory/app/ChildHomeActivity.java [hunk 1]
```diff
@@ -81,6 +81,8 @@ protected void onCreate(Bundle savedInstanceState) {
             @Override
             public void onTabSelected(TabLayout.Tab tab) {
                 tab.setIcon(tabSelectedIcons[tab.getPosition()]);
+                int i = tab.getPosition();
+                if (i == 2) fragmentList.get(i).onResume();
```
Comment (isabelleleeson) [inline android/app/src/main/java/com/chorestory/app/ChildHomeActivity.java]: can leave the quests page then come back to make sure quests refresh

File: android/app/src/main/java/com/chorestory/adapter/ChildRecyclerViewAdapter.java [hunk 2]
```diff
@@ -25,9 +25,16 @@ public ChildRecyclerViewAdapter(List<Children> childrenList) {
         for (Children child : childrenList) {
             this.childrenList.add(new ChildRecyclerViewItem(child));
         }
-
         // Sort the list of Children in descending order of level and exp
         Collections.sort(this.childrenList);
+
```
Comment (isabelleleeson) [inline android/app/src/main/java/com/chorestory/adapter/ChildRecyclerViewAdapter.java]: list of children: when index is even, use knight, when odd use joker

File: android/app/src/main/java/com/chorestory/app/ParentCreateClanActivity.java [hunk 3]
```diff
@@ -71,7 +90,6 @@ public void onClick(View v) {
                 clanName = clanNameEditText.getText().toString();
                 name = nameEditText.getText().toString();
                 password = passwordEditText.getText().toString();
-                picture = R.drawable.king_color; // TODO: Let the user select this when the option is created
```
Comment (keller00) [inline android/app/src/main/java/com/chorestory/app/ParentCreateClanActivity.java]: Shouldn't this be `picture = picture`?

Comment (isabelleleeson) [reply android/app/src/main/java/com/chorestory/app/ParentCreateClanActivity.java]: I set  `picture = R.drawable.king_color;` as default in onCreate, then it updates if user clicks on the other one. this is correct

Comment (keller00) [reply android/app/src/main/java/com/chorestory/app/ParentCreateClanActivity.java]: Okay, nevermind I see it!

**Pull request merged**